pull_request_template.md

```diff
@@ -0,0 +1,26 @@
+## Link to JIRA ticket
```

Comment:
Is this file necessary? :o

Reply:
It's the template that pre-populates the new PR description. I added it to make it align with the other LBH repos.